app/build.gradle

```diff
@@ -172,6 +172,12 @@ dependencies {
     // Facebook Login
     implementation "com.facebook.android:facebook-login:$facebook_login_version"
 
+    // Retrofit
+    implementation 'com.squareup.retrofit2:retrofit:2.6.2'
```

Comment:
maybe you can move the version number to buildscript like "$facebook_login_version"
`buildscript {
ext.retrofit_version = '2.6.2'
}`